solidity/package.json

```diff
@@ -43,7 +52,14 @@
     "@truffle/hdwallet-provider": "^1.0.25",
     "bn-chai": "^1.0.1",
     "chai": "^4.2.0",
+    "eslint": "^6.8.0",
+    "eslint-config-keep": "github:keep-network/eslint-config-keep",
+    "eslint-config-prettier": "^6.10.1",
```

Comment:
Burn it please :) `eslint-config-keep` should handle this. I realize it doesn't currently deal with the Solidity plugin, but we still need to explicitly include the prettier dependency here anyway.

My suggestion:
- Kill `eslint-config-prettier`. Let `eslint-config-keep#0.3.0` handle the JS side of prettier. You still need the prettier dependency below for this, but you lose the explicit `prettier` invocation for `lint:js` and `lint:fix:js`.
- Continue to pull in `prettier-plugin-solidity` and run prettier in the `:sol` scripts.

I also want to see if I can pull prettier into solium's config, but that seems unlikely. There may be room to set something up with our core eslint config and Solidity though, TBD.